config/samples/ccruntime/default/kustomization.yaml

```diff
@@ -8,10 +8,9 @@ resources:
 
 images:
 - name: quay.io/confidential-containers/reqs-payload
-  newTag: latest
+  newTag: 12e8fbc72e927f9d684efe2e1214937f8b0544fc
 - name: quay.io/kata-containers/kata-deploy
-  newName: quay.io/kata-containers/kata-deploy-ci
-  newTag: kata-containers-latest
+  newTag: v3.9.0
```

Comment:
This tag isn't correct - there is no `v`

Reply:
@stevenhorsman ,  you are right. I just corrected that. Thanks!